src/components/orders/AmountsDisplay/styled.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components'
+
+import TokenImg from 'components/common/TokenImg'
+
+export const Wrapper = styled.div`
+  display: grid;
+  row-gap: 1rem;
+  justify-items: start;
+  align-items: center;
+  grid-template-columns: 10rem auto;
+`
+
+export const RowTitle = styled.span`
+  ::before {
+    content: '▶';
+    margin-right: 0.5rem;
+    color: ${({ theme }): string => theme.icon};
+    font-size: 0.75rem;
+  }
+
+  font-weight: ${({ theme }): string => theme.fontWeightBold};
+`
+
+export const RowContents = styled.span`
+  display: flex;
+  align-items: center;
+
+  & > :first-child {
+    margin: 0 0.5rem 0 0;
+  }
+
+  & > * {
```

Comment:
I'd prevent the wildcard at all times, because of performance and unexpected matches. 

I'd be as specific in the structure as possible (like you did with & > :first-child).

Reply:
In this case, what would be the best way to select all children without using wild cards?
I assume an inverted selector would be equally bad, such as: `& > :not(:first-child)` ?

Reply:
My first thought would be to try to use a single HTML element you could target. If not possible specify all selectors?